addons/ofxOpenCv/addon_config.mk

```diff
@@ -61,7 +61,7 @@ common:
 	# ADDON_LIBS_EXCLUDE =
 
 linux64:
-    ADDON_PKG_CONFIG_LIBRARIES = opencv4 harfbuzz
+    ADDON_PKG_CONFIG_LIBRARIES = opencv harfbuzz
```

Comment:
this should be kept as opencv4 you might have commited it after the install_dependencies script changes it cause opencv4 is not yet avaliable on the pi.

reverting it to opencv4 should fix some of the issues in other platforms

Reply:
ooh, I was wondering how that file got changed without me touching it